jenkins_pipelines/environments/common/pipeline-pull-request-light.groovy

```diff
@@ -0,0 +1,208 @@
+                    if(must_test) {
+                        sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:core'"
+                        sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:reposync'"
+                    }
+                }
+            }
+            stage('Core - Initialize clients') {
+                ws(environment_workspace){
+                    if(must_test) {
+                        namespace = rake_namespace
+                        sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake ${namespace}:init_clients'"
+                    }
+                }
+            }
+            stage('Secondary features') {
+                ws(environment_workspace){
+                    if(must_test && ( params.functional_scopes || run_all_scopes) ) {
+                        def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd '${secondary_exports} cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
+                        def statusCode2 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd '${secondary_exports} cd /root/spacewalk/testsuite; rake ${rake_namespace}:secondary_parallelizable'", returnStatus:true
+                        sh "exit \$(( ${statusCode1}|${statusCode2} ))"
+                    }
+                }
+                tests_passed = true
+            }
+        }
+        finally {
+            stage('Clean up lockfiles') {
+                if(running_same_pr == "no"){
+                    sh(script: "rm -f ${env.suma_pr_lockfile}")
+                }
+                if(environment_workspace){
+                    ws(environment_workspace){
+                        if (env.env_file) {
+                            if (tests_passed || !deployed){
+                                println("Unlock environment")
+                                sh "rm -f ${env_file}*"
+                            } else {
+                                println("Keep the environment locked for 24 hours so you can debug")
+                                sh "echo \"rm -f ${env_file}*\" | at now +24 hour"
+                                sh "echo keep:24h >> ${env_file}.info"
+                                sh "python3 ${WORKSPACE}/product/susemanager-utils/testing/automation/run-command-in-server.py --command=\"chmod 755 /tmp/set_custom_header.sh;/tmp/set_custom_header.sh -e ${env_number} -m ${email_to} -t 24\" --username=\"root\" --password=\"linux\" -v -i suma-pr${env_number}-srv.mgr.prv.suse.net"
```

Comment:
Be caution with this domain, we might need to parametrize it too.
Depending if our environments are deployed in NUE or PRV.